article/_posts/2019-07-29-Dont-Worry-About-The-Weather.md

```diff
@@ -0,0 +1,101 @@
+
+- Training input image adaptors by using the *off-the-shelf* computer vision models to generate a supervisory signal.
+
+- Enabling online learning of new, unseen domains by leveraging the unsupervised data generation pipeline along with domains on which the data generation models have already been trained.
+
+- Showing that training multiple light-weight adapter modules is better than training monolithic computer vision models that are invariant to input distributions.
+
+
+# Learning Condition-Dependent Representations
+
+![](/article/images/Weather/Gan.png)
+
+![](/article/images/Weather/archi.png)
+
+## Synthetic Multi-Condition Data
+
+They select a reference sequence of images: daytime, clear, and overcast.
+Besides, they join some traversals with severe conditions: night, rain, snow, etc.
+Along with a cycle-consistency architecture, GAN to train generative models that can apply style transfer to the reference condition in order to create a number of synthetic sequences that maintain the structure and geometry of the reference condition.
+
+Loss adversarial:
+
+![](/article/images/Weather/loss_adv_1.png)
+![](/article/images/Weather/loss_adv_2.png)
+
+Loss for discriminators:
+
+![](/article/images/Weather/loss_disc_1.png)
+![](/article/images/Weather/loss_disc_2.png)
+
+Reconstruction and Generator Loss:
+
+![](/article/images/Weather/loss_rec.png)
+![](/article/images/Weather/loss_gen.png)
+
+## Input Adapters
+
+The second step, in this approach, is to use the data generated in the previous step to train a bank of adapters that preprocess the input images. Like they follow a distribution similar to that of the training sets used to train the bank of tasks.
+
+
+The adaptors input is a 3-channel RGB image, while the output is a 3-channel image compatible with the inputs of many well-known models (semantic segmentation, object detection, depth estimation, etc.).
+This configuration provides a light-weight solution that is easy to train using labeled data, with reduced storage requirements and use a small amount of memory.
+
+![](/article/images/Weather/loss_adapters.png)
+
+## Domain Classifier
+
+![](/article/images/Weather/classifier.png)
+
+They employ a domain classifier $$D$$ to select the most suitable input adapter $$F_k$$ that enables optimal performance on the input images with the $$k^{th}$$ condition.
+Given an input image $$I_A$$ and a domain label $$t$$, the goal is to find the parameters of the classifier $$D$$ that minimizes the cross-entropy between the output of the classifier and the target label $$t$$.
+
+## Online Learning
+
+![](/article/images/Weather/onlinelearning.png)
+
+The previous subsections can extend to incremental unsupervised, online learning of new unseen domains without requiring any significant modifications to the existing system.
+This processed used to:
```

Comment:
This process is used to ?

But the bullet points do not seem to follow a pattern following the above sentence, so they paragraph should be reworked.

Reply:
Done in 07b72cb.